packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -152,6 +175,12 @@ export const { command, description, builder, handler } =
           'Use when you want to generate a cell for a list of the model name.',
         type: 'boolean',
       },
+      query: {
+        default: '',
+        description:
+          'Use to enforce a specific query name within the generated cell.',
```

Comment:
```suggestion
          'Use to enforce a specific query name within the generated cell - must be unique',
```

Reply:
Definitely needed, I added this exact change. Would have committed it directly from here but it wasn't happy with it.